packages/web3torrent/src/clients/payment-channel-client.ts

```diff
@@ -81,17 +79,17 @@ export class PaymentChannelClient {
     });
   }
 
+  async initialize() {
```

Comment:
I'm not sure if there's a better naming convention. Right now it's slightly confusing that there's an `initialize` and an `enable` on the payment channel client.